.github/CONTRIBUTING.md

```diff
@@ -52,6 +52,8 @@ If you're not sure what any of that means, check out Thinkful's [GitHub Pull Req
 
 - Make sure the code you submit compiles and runs without issues. When we set up unit tests and continuous integration we also expect that the pull request should pass all tests.
 
+- Use [closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords)  in the appropriate section of our Pull Request template where applicable.
```

Comment:
Not sure if it has impact... Just want to be sure.. And good work 👍 

```suggestion
- Use [closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords) in the appropriate section of our Pull Request template where applicable.
```

Reply:
It's a good suggestion and I hope it will have an impact.